tests/cps-files/lib/cps/full.cps

```diff
@@ -0,0 +1,53 @@
+{
+    "name": "full",
+    "cps_version": "0.10.0",
+    "version": "1.0.1",
+    "components": {
+        "sample0": {
+            "type": "archive",
+            "compile_flags": [
+                "-fsentinal"
+            ],
+            "includes": {
+                "c": [
+                    "/err"
+                ]
+            },
+            "defines": {
+                "c": [
+                    "-DFAIL"
+                ]
+            },
+            "location": "fake"
+        },
+        "fullSample": {
+            "type": "archive",
```

Comment:
...but the locations are `.so`s. Probably this should be `dylib`?

Reply:
Make sense